### PR TITLE
Updated README.md for Kube-UI Addon Installation

### PR DESCRIPTION
**Pull request by ankitforcode**
Updated README.md for Kube-UI Addon Installation

### CODE REVIEW

File: multi-node/aws/README.md [hunk 1]
```diff
@@ -24,6 +24,32 @@ $ aws s3 cp --recursive --acl=public-read artifacts/ s3://<bucket>/
 
 Then, simply create a cluster using `artifactURL: https://<bucket>.s3.amazonaws.com`.
 
+### Accessing the Kubernetes Cluster Locally
+
+```
+$ ssh -f -nNT -L 8080:127.0.0.1:8080 -i <private-key> core@<cluster-external-dns-name>
```
Comment (aaronlevy) [inline multi-node/aws/README.md]: I'd prefer that we document using `kubectl proxy` as the method for tunneling to apiserver. The above will also work, but the contract we are maintaining is that "apiserver is accessible via config options in a kubeconfig". With that contract, `kubectl proxy` should be able to handle tunnel/auth for us.

File: multi-node/aws/README.md [hunk 2]
```diff
@@ -24,6 +24,32 @@ $ aws s3 cp --recursive --acl=public-read artifacts/ s3://<bucket>/
...
+```
+
+### Kube UI Setup
+
+Once the Kubernetes Cluster API is available locally the Kube-UI can be deployed using the below.
+
+```
+$ git clone https://github.com/kubernetes/kubernetes
+$ cd kubernetes && git checkout v1.1.1
+$ kubectl create -f kubernetes/cluster/addons/kube-ui/
```
Comment (aaronlevy) [inline multi-node/aws/README.md]: Can we be explicit about each object we are creating? e.g:

```
kubectl create -f kubernetes/cluster/addons/kube-ui/kube-ui-rc.yaml
kubectl create -f kubernetes/cluster/addons/kube-ui/kube-ui-svc.yaml
```

File: multi-node/aws/README.md [hunk 3]
```diff
@@ -24,6 +24,32 @@ $ aws s3 cp --recursive --acl=public-read artifacts/ s3://<bucket>/
...
+
+```
+# Load kube-UI in browser at: http://127.0.0.1:8080/ui
```
Comment (aaronlevy) [inline multi-node/aws/README.md]: This shouldn't be a H1 header.

**Comment by aaronlevy** [comment]
I also don't think the AWS readme is the ideal place for this documentation. Either adding it to a "Optional Addons" section in Documentation/deploy-addons.md, or possibly starting a new doc for "deploy-optional-addons.md"

/cc @robszumski

**Comment by robszumski** [comment]
Yes, I like @aaronlevy's idea of an `optional-addons.md` page. We can link to it from the existing page as a possible next step, like the guestbook is now.

File: multi-node/aws/README.md (hunk 2 above)
Comment (mvoropaiev) [inline multi-node/aws/README.md]: And if we already cd into kubernetes directory in step 2 the create command should probably look like `kubectl create -f ./cluster/addons/...`

**Comment by bcwaldon** [comment]
@ankitforcode any progress here?

**Pull request closed without merging**